decidim-assemblies/lib/decidim/assemblies/engine.rb

```diff
@@ -49,24 +50,8 @@ class Engine < ::Rails::Engine
         end
       end
 
-      initializer "decidim_assemblies.menu" do
-        Decidim.menu :menu do |menu|
-          menu.add_item :assemblies,
-                        I18n.t("menu.assemblies", scope: "decidim"),
-                        decidim_assemblies.assemblies_path,
-                        position: 2.2,
-                        if: OrganizationPublishedAssemblies.new(current_organization, current_user).any?,
-                        active: :inclusive
-        end
 
-        Decidim.menu :home_content_block_menu do |menu|
-          menu.add_item :assemblies,
-                        I18n.t("menu.assemblies", scope: "decidim"),
-                        decidim_assemblies.assemblies_path,
-                        position: 20,
-                        if: OrganizationPublishedAssemblies.new(current_organization, current_user).any?,
-                        active: :inclusive
-        end
+        Decidim::Assemblies::Menu.register_engine_menu!
```

Comment:
You may have removed too much ... 
```suggestion
      initializer "decidim_assemblies.menu" do
        Decidim::Assemblies::Menu.register_engine_menu!
```

Reply:
When I start removing, I just can't stop